### PR TITLE
fix: Fixed build errors

### PR DESCRIPTION
**Pull request by magnuh**

### CODE REVIEW

File: src/attention/utils/helpers.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { ReferenceElement } from "@floating-ui/core";
+import { ReferenceElement } from "@floating-ui/dom";
 import { computePosition, flip, offset, shift, arrow } from "@floating-ui/dom";
```
Comment (AnnaRybkina) [inline src/attention/utils/helpers.ts]:
```suggestion
import { computePosition, flip, offset, shift, arrow, ReferenceElement } from "@floating-ui/dom";
```

**Review by BalbinaK — APPROVED** [review]
Are you sure it wasn't that `@floating-ui/core` package that should be installed? Just checking because we should maybe fix this in [@fabric-ds/core](https://github.com/fabric-ds/core/tree/main/src/attention/utils), too.

**Comment by magnuh** [comment]
> Are you sure it wasn't that `@floating-ui/core` package that should be installed? Just checking because we should maybe fix this in [@fabric-ds/core](https://github.com/fabric-ds/core/tree/main/src/attention/utils), too.

floating-ui/core is imported in floating-ui/dom so I guess it just magically works with npm? 🤷  I think we should leave fabric as it is... and if any problems with the new setup we will notice when using warp-ds/core in the other repos? 🙂

**Review by AnnaRybkina — APPROVED** [review]

File: src/slider/utils/handlers.ts [hunk 2]
```diff
@@ -67,11 +67,11 @@ export function createHandlers({
     if (!validKeyCodes.includes(key)) return;
     e.preventDefault();
     if (
-      [validKeys.left, validKeys.right, validKeys.up, validKeys.down].includes(
+        ([validKeys.left, validKeys.right, validKeys.up, validKeys.down] as ElementType<typeof validKeyCodes>[]).includes(
         key
       )
     ) {
-      const direction = [validKeys.right, validKeys.up].includes(key) ? 1 : -1;
+      const direction = ([validKeys.right, validKeys.up] as ElementType<typeof validKeyCodes>[]).includes(key) ? 1 : -1;
```
Comment (AnnaRybkina) [inline src/slider/utils/handlers.ts:74]: fwiw I'm not a big fan of casting. Thinking that validKeys should have a proper type already 🤔 . But it doesn't feel like something that is needed to be fixed on another level now 👍

**Pull request merged**